pkg/apis/scorecard/formatting.go

```diff
@@ -15,15 +15,11 @@
 package scorecard
 
 import (
-	scapiv1alpha1 "github.com/operator-framework/operator-sdk/pkg/apis/scorecard/v1alpha1"
 	scapiv1alpha2 "github.com/operator-framework/operator-sdk/pkg/apis/scorecard/v1alpha2"
 )
 
-var _ ScorecardFormatter = &scapiv1alpha1.ScorecardOutput{}
-var _ ScorecardFormatter = &scapiv1alpha2.ScorecardOutput{}
+var _ Formatter = &scapiv1alpha2.ScorecardOutput{}
```

Comment:
This type check should be in the `v1alpha2` package.

Reply:
that entire interface I think can be removed at this point actually since we only have v1alpha2 left.